.pipelines/azure_pipeline_prod.yaml

```diff
@@ -14,33 +14,32 @@ pr:
     include:
     - ci_prod
 
-pool:
-  name: Azure-Pipelines-CI-Prod-EO
-
 variables:
-  armServiceConnectionName: 'ci-1es-acr-connection-prod'
-  subscription: '30c56c3a-54da-46ea-b004-06eb33432687'
-  containerRegistry: 'containerinsightsbuild'
-  repoImageNameLinux: '${{ variables.containerRegistry }}.azurecr.io/official/linux'
-  repoImageNameWindows: '${{ variables.containerRegistry }}.azurecr.io/official/windows'
+  armServiceConnectionName: 'ci-1es-acr-connection'
+  subscription: '9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb'
+  containerRegistry: 'containerinsightsprod'
+  repoImageName: '${{ variables.containerRegistry }}.azurecr.io/public/azuremonitor/containerinsights/cidev'
   IS_PR: $[eq(variables['Build.Reason'], 'PullRequest')]
 
 jobs:
 - job: common
   pool:
-    name: Azure-Pipelines-CI-Prod-EO
+    name: Azure-Pipelines-CI-Test-EO
   steps:
   - bash: |
       commit=$(git rev-parse --short HEAD)
       datetime=$(date +'%m%d%Y')
-      linuxImagetag=ciprod-"$datetime"-"$commit"
-      windowsImageTag=win-ciprod-"$datetime"-"$commit"
+      linuxImagetag="$datetime"-"$commit"
+      windowsImageTag=win-"$datetime"-"$commit"
       echo "##vso[task.setvariable variable=linuxImagetag;isOutput=true]$linuxImagetag"
       echo "##vso[task.setvariable variable=windowsImageTag;isOutput=true]$windowsImageTag"
 
       cd $(Build.SourcesDirectory)/deployment/multiarch-agent-deployment/ServiceGroupRoot/Scripts
```

Comment:
is it needed for the new pipeline yaml?

Reply:
What do you mean? The tags will be used in docker build right?

Reply:
not sure why this comment is showing on this line. will place it at the right place